src/pages/Campaign/LeaderboardLayout.tsx

```diff
@@ -0,0 +1,344 @@
+            <LeaderboardTableHeaderItem align="right">
+              <Trans>Rewards</Trans>
+            </LeaderboardTableHeaderItem>
+          )}
+        </LeaderboardTableHeader>
+        {searchedLeaderboard
+          .slice((currentPage - 1) * CAMPAIGN_ITEM_PER_PAGE, currentPage * CAMPAIGN_ITEM_PER_PAGE)
+          .map((data, index) => (
+            <LeaderboardTableBody
+              key={index}
+              showRewards={showRewards}
+              style={{
+                background:
+                  data.rank === 1
+                    ? 'linear-gradient(90deg, rgba(255, 204, 102, 0.25) 0%, rgba(255, 204, 102, 0) 54.69%, rgba(255, 204, 102, 0) 100%)'
+                    : data.rank === 2
+                    ? 'linear-gradient(90deg, rgba(224, 224, 224, 0.25) 0%, rgba(224, 224, 224, 0) 54.69%, rgba(224, 224, 224, 0) 100%)'
+                    : data.rank === 3
+                    ? 'linear-gradient(90deg, rgba(255, 152, 56, 0.25) 0%, rgba(255, 152, 56, 0) 54.69%, rgba(255, 152, 56, 0) 100%)'
+                    : 'transparent',
+                padding: data.rank <= 3 ? '16px 20px' : '20px',
+              }}
+            >
+              <LeaderboardTableBodyItem
+                align="center"
+                style={{ width: (rankWidth === Infinity ? 33 : rankWidth) + 'px', maxHeight: '24px' }}
+              >
+                {data.rank === 1 ? (
+                  <img src={Gold} style={{ minWidth: '18px' }} alt="" />
+                ) : data.rank === 2 ? (
+                  <img src={Silver} style={{ minWidth: '18px' }} alt="" />
+                ) : data.rank === 3 ? (
+                  <img src={Bronze} style={{ minWidth: '18px' }} alt="" />
+                ) : data.rank !== undefined ? (
+                  data.rank
+                ) : null}
+              </LeaderboardTableBodyItem>
+              <LeaderboardTableBodyItem>{getShortenAddress(data.address, above1200)}</LeaderboardTableBodyItem>
+              <LeaderboardTableBodyItem align="right">
+                {formatNumberWithPrecisionRange(data.point, 0, 2)}
+              </LeaderboardTableBodyItem>
+              {showRewards && (
+                <LeaderboardTableBodyItem align="right">
+                  {data.rewardAmount} {data.token}
+                </LeaderboardTableBodyItem>
+              )}
+            </LeaderboardTableBody>
+          ))}
+      </LeaderboardTable>
+      <Pagination
+        onPageChange={setCurrentPage}
+        totalCount={searchedLeaderboard.length}
+        currentPage={currentPage}
+        pageSize={CAMPAIGN_ITEM_PER_PAGE}
+        style={{ padding: '0' }}
+      />
+    </LeaderboardContainer>
+  )
+}
+
+const LeaderboardContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+`
+
+const RefreshTextAndSearchContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+  ${css`
```

Comment:
why do you need css`` here. I usually use css content directly inside media width